webapp/commonComponents/hooks/useNodeDef.js

```diff
@@ -2,7 +2,5 @@ import { useSelector } from 'react-redux'
 
 import * as NodeDefState from '@webapp/loggedin/surveyViews/nodeDef/nodeDefState'
 
-/**
- * Gets the nodeDef from the survey views node def state.
- */
+// Gets the nodeDef from the survey views node def state.
```

Comment:
also this doesn't look like a generic hook, or is it?

Reply:
uhm, right, it's used only by NodeDefView or its inner components, I'll remove it